src/components/Dialog/Dialog.tsx

```diff
@@ -0,0 +1,167 @@
+      >
+        {label}
+      </Button>
+    </>
+  );
+};
+
+DialogClose.displayName = "DialogClose";
+Dialog.Close = DialogClose;
+
+// Dialog Content
+const overlayShow = keyframes({
+  "0%": { opacity: 0 },
+  "100%": { opacity: 1 },
+});
+
+const contentShow = keyframes({
+  "0%": { opacity: 0, transform: "translate(-50%, -48%) scale(.96)" },
+  "100%": { opacity: 1, transform: "translate(-50%, -50%) scale(1)" },
+});
+
+const DialogOverlay = styled(RadixDialog.Overlay)`
+  background-color: ${({ theme }) => theme.click.dialog.color.opaqueBackground.default};
+  position: fixed;
+  inset: 0;
+  animation: ${overlayShow} 150ms cubic-bezier(0.16, 1, 0.3, 1);
+`;
+
+const ContentArea = styled(RadixDialog.Content)`
+  background: ${({ theme }) => theme.click.dialog.color.background.default};
+  border-radius: ${({ theme }) => theme.click.dialog.radii.all};
+  padding: ${({ theme }) => theme.click.dialog.space.y}
+    ${({ theme }) => theme.click.dialog.space.x};
+  box-shadow: ${({ theme }) => theme.click.dialog.shadow.default};
+  border: 1px solid ${({ theme }) => theme.click.global.color.stroke.default};
+  width: 75%;
+  max-width: 670px;
+  position: fixed;
+  top: 25%;
+  margin-top: 20%;
+  left: 50%;
+  max-height: 75%;
+  overflow: auto;
+  transform: translate(-50%, -50%);
+  animation: ${contentShow} 150ms cubic-bezier(0.16, 1, 0.3, 1);
+  outline: none;
+`;
+
+const TitleArea = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  min-height: ${({ theme }) => theme.sizes[9]}; // 32px
+`;
+
+const CloseButton = styled(EmptyButton)`
+  padding: ${({ theme }) => theme.click.button.iconButton.sm.space.y}
+    ${({ theme }) => theme.click.button.iconButton.sm.space.x};
+  background: ${({ theme }) =>
+    theme.click.button.iconButton.color.primary.background.default};
+  border-radius: ${({ theme }) => theme.click.button.iconButton.radii.all};
+
+  &:hover {
+    background: ${({ theme }) =>
+      theme.click.button.iconButton.color.primary.background.hover};
+  }
+`;
+
+interface DialogContentProps extends RadixDialog.DialogContentProps {
+  title: string;
+  showClose?: boolean;
+  forceMount?: true;
+  container?: HTMLElement | null;
+}
+
+const DialogContent = ({
+  title,
+  children,
+  showClose,
+  forceMount,
+  container,
+}: DialogContentProps) => {
+  return (
+    <RadixDialog.Portal
+      forceMount={forceMount}
+      container={container}
+    >
+      <DialogOverlay />
+      <ContentArea>
+        <TitleArea>
+          <Title
+            size="xl"
+            type="h2"
+          >
+            {title}
+          </Title>
+          {showClose && (
+            <CloseButton
+              as={RadixDialog.Close}
+              asChild
+            >
+              <Icon
+                name="cross"
+                size="lg"
+              />
+            </CloseButton>
+          )}
+        </TitleArea>
+        <Spacer />
+        {children}
```

Comment:
do we plan to have the `Actions` area customizable?

I think we can define the styles here and avoid making the users to always style the Cancel/Confirm actions

If not, we can create two dialogs components, one that is simpler and less customizable and that can be reused for 80% of the cases and one more flexible to accommodate specific needs. What do you think?